lambda_functions/v1/requirements/requirements.txt

```diff
@@ -1,8 +1,8 @@
 #Update this date to trigger update of layers: 20200807
 Flask==1.1.2
 Werkzeug==1.0.1
-pyjwt==2.0.0
+pyjwt==1.7.1
```

Comment:
I believe you can take this out entirely now that you have dependency in opg-data. Tried it without locally yesterday and it worked without.